package.json

```
@@ -69,7 +69,7 @@
   "resolutions": {
     "connected-next-router/react-redux": "8.0.7",
     "react-beautiful-dnd/react-redux": "8.0.7",
-    "@prismicio/types-internal": "2.8.0",
+    "@prismicio/types-internal": "2.9.0-alpha.0",
```

To be applied when package is released

```suggestion
    "@prismicio/types-internal": "2.9.0",
```